web-app/src/main/webapp/script.js

```diff
@@ -59,8 +61,25 @@ function showMessageOnInfoWindow(message, position, map, infoWindow) {
 window.onload = () => {
   document.getElementById("form-container").style.display = "none";
   document.getElementById('report-button').addEventListener('click', showReportForm);
+  loadPoliceReports();
 };
 
 function showReportForm() {
   document.getElementById("form-container").style.display = "block";
 }
+
+async function loadPoliceReports() {
+  const FILE_NAMES = ['2019_12_london', '2020_01_london', '2020_02_london', '2020_03_london', '2020_04_london', '2020_05_london']
+  for (let i = 0; i < FILE_NAMES.length; i++) {
+    const data = await fetch('../data/' + FILE_NAMES[i] + '.json');
+    const report = await data.json();
+    report.forEach((report) => {
+      new google.maps.Marker({
+        position: {
+          lat: parseFloat(report.Latitude),
```

Comment:
nit: does it work if you just use the Number constructor? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/Number